content/02.introduction.md

```diff
@@ -154,14 +154,17 @@ The mechanism behind these insults has been suggested to be related to inflammat
 Individual outcomes have varied significantly, and reported hospitalization rates have been wide-ranging based on age and location, with various underlying health conditions being disproportionately reported [@doi:10.15585/mmwr.mm6915e3].
 Outcomes in patients that are hospitalized have generally been poor, with rates of admission to the intensive care unit (ICU) upwards of 15% in both Wuhan, China and Italy [@doi:10.1056/nejmoa2002032; @doi:10.1001/jama.2020.2648; @doi:10.1001/jama.2020.4031].
 
-Though older populations are generally considered the most vulnerable, pediatric infections are also a concern.
+The results of COVID-19 infection can vary greatly among pediatric patients as well as adults. 
 [@doi:10.1001/jamapediatrs.2020.1467] reviewed studies on children infected with COVID-19 during the early months of COVID-19 in China.
 Most of the children reviewed had mild symptoms such as fever, dry cough, fatigue, nasal congestion and/or runny nose or were asymptomatic.
 Gastrointestinal symptoms such as vomiting or diarrhea were occasionally reported.
-Radiologic findings were not always mentioned, but when mentioned they included bronchial thickening, ground-glass opacities, and/or inflammatory lesions.
-The results of COVID-19 infection can vary greatly among pediatric patients as well as adults.
-In particular, some children have experienced a severe inflammatory syndrome after COVID-19 infection, similar in some respects to Kawasaki disease or to Kawasaki disease shock syndrome [@doi:10.1093/jpids/piaa069;@doi:10.1001/jama.2020.10369; @doi:10.1016/j.ajem.2020.05.117].
-Heart failure can also occur [@doi:10.1161/CIRCULATIONAHA.120.048360].
+Radiologic findings were not always mentioned, but when mentioned they included bronchial thickening, ground-glass opacities, and/or inflammatory lesions. Neurological symptoms have also been reported [@doi:10.1001/jamaneurol.2020.2687].
+Generally, most cases of COVID-19 in children are not severe.  
+However, serious complications, and in rare cases deaths, have occurred
+[@url: https://www.cnn.com/2020/04/21/us/detroit-girl-dies-coronavirus/index.html; @doi:10.15585/mmwr.mm6914e4.]. 
```

Comment:
```suggestion
However, serious complications, and in rare cases deaths, have occurred [@https://www.cnn.com/2020/04/21/us/detroit-girl-dies-coronavirus/index.html; @doi:10.15585/mmwr.mm6914e4]. 
```

Using the CNN story to support this story seems a little anecdotal.  What about replacing the CNN reference with this systematic review:
COVID-19 in 7780 pediatric patients: A systematic review
https://doi.org/10.1016/j.eclinm.2020.100433

Reply:
I agree!  I was not very happy about citing a TV news station in an academic paper, but couldn't quite decide on anything better.  I will make the change.